ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/MacOSX.yml

```diff
@@ -0,0 +1,21 @@
+---
+##########
+# macOS #
+##########
+
+# Command Build Tool Packages
+
+Build_Tool_Packages:
+  - gnu-tar
+  - wget
+
+Build_Tool_Casks:
+  - adoptopenjdk8
+
+Test_Tool_Packages:
+  - mercurial
```

Comment:
Do we need mercurial on here?

Reply:
it's a test dependency apparently

Reply:
@smlambert can you confirm this?

Reply:
I was just going on what was here: https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/Ubuntu.yml#L104